lib/streamlit/web/cache_storage_manager_config.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from streamlit.runtime.caching.storage import CacheStorageManager
+from streamlit.runtime.caching.storage.local_disk_cache_storage import (
+    InMemoryWrappedLocalDiskCacheStorageManager,
+)
+
+
+def get_cache_storage_manager() -> CacheStorageManager:
```

Comment:
Nit: this unconditionally creates a new CacheStorageManager, but the name makes it sound like it just gets an existing one. Can we change it to `create_default_cache_storage_manager` or something?